pydocx/__main__.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import (
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+import sys
+import logging
+
+from pydocx.parsers.Docx2Html import Docx2Html
+from pydocx.parsers.Docx2Markdown import Docx2Markdown
+
+
+def convert(parser_type, docx_path, output_path):
+    if parser_type == '--html':
+        output = Docx2Html(docx_path).parsed
+    elif parser_type == '--markdown':
+        output = Docx2Markdown(docx_path).parsed
+    else:
+        print('Only valid parsers are --html and --markdown')
+        return 2
+    with open(output_path, 'wb') as f:
+        f.write(output.encode('utf-8'))
+    return 0
+
+
+def usage():
+    print('Usage: pydocx [--html|--markdown] input.docx output')
```

Comment:
Conventionally, brackets indicate optional arguments. Required arguments are not inside brackets.

`Usage: pydocx --html|--markdown input.docx output`